### PR TITLE
Feature/108388

### PR DESCRIPTION
**Pull request by Jailsvs**
[AB#108388](https://dev.azure.com/amcomgov/c1dcf343-60ee-40b6-a30a-3b9c12c6d220/_workitems/edit/108388)

### CODE REVIEW

**Comment by luizhpriotto** [comment]
✅ Successfully linked to Azure Boards work item(s):
- [Task 108388](https://dev.azure.com/amcomgov/c1dcf343-60ee-40b6-a30a-3b9c12c6d220/_workitems/edit/108388)

**Comment by luizhpriotto** [comment]
✅ Successfully linked to Azure Boards work item(s):
- [Task 108388](https://dev.azure.com/amcomgov/c1dcf343-60ee-40b6-a30a-3b9c12c6d220/_workitems/edit/108388)

File: src/SME.SGP.Aplicacao/Queries/ConsolidacaoFrequenciaMensal/ObterConsolidacoesFrequenciaAlunoMensalInsuficientes/ObterConsolidacoesFrequenciaAlunoMensalInsuficientesQuery.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        public int AnoLetivo { get; set; }
+    }
+
+    public class ObterConsolidacoesFrequenciaAlunoMensalInsuficientesQueryValidator : AbstractValidator<ObterConsolidacoesFrequenciaAlunoMensalInsuficientesQuery>
+    {
+        public ObterConsolidacoesFrequenciaAlunoMensalInsuficientesQueryValidator()
+        {
+            RuleFor(c => c.UeId)
+                .GreaterThan(0)
+                .WithMessage("O Id da Ue percisa ser informado");
```
Comment (viniciusnyari) [inline src/SME.SGP.Aplicacao/Queries/ConsolidacaoFrequenciaMensal/ObterConsolidacoesFrequenciaAlunoMensalInsuficientes/ObterConsolidacoesFrequenciaAlunoMensalInsuficientesQuery.cs]: Colocar um sufixo aqui

File: src/SME.SGP.Dados/Repositorios/RepositorioConsolidacaoFrequenciaAlunoMensal.cs [hunk 2]
```diff
@@ -67,5 +67,23 @@ public async Task RemoverConsolidacaoAluno(long consolidacaoId)
...
+                                  u.nome as ue, u.tipo_escola as tipoEscola, d.abreviacao as dre,
+                                  cfam.percentual as 	, cfam.mes, cfam.aluno_codigo as alunoCodigo
+                           from consolidacao_frequencia_aluno_mensal cfam
+                           inner join turma t on t.id = cfam.turma_id  
+                           inner join ue u on u.id = t.ue_id 
+                           inner join dre d on d.id = u.dre_id 
+                           where t.ue_id = @ueId
+                                 and cfam.mes = @mes
+                                 and t.ano_letivo = @anoLetivo
+                                 and ((t.modalidade_codigo in(3,5,6) and cfam.percentual < 75) or
```
Comment (viniciusnyari) [inline src/SME.SGP.Dados/Repositorios/RepositorioConsolidacaoFrequenciaAlunoMensal.cs]: Tá brabo aqui sem constante

**Review by viniciusnyari — APPROVED** [review]

**Pull request merged**